src/imitation/scripts/train_preference_comparisons.py

```diff
@@ -177,6 +181,18 @@ def train_preference_comparisons(
         seed=_seed,
         custom_logger=custom_logger,
     )
+    if active_selection:
+        preference_predictor = preference_comparisons.PreferencePredictor(
```

Comment:
Related to a previous comment in `preference_comparisons.py`: I find it odd that we have to create `PreferencePredictor` here but only for `ActiveSelectionFragmenter`